website/project/model.py

```diff
@@ -2119,7 +2119,11 @@ def fork_node(self, auth, title='Fork of '):
                 if forked_node is not None:
                     forked.nodes.append(forked_node)
 
-        forked.title = title + forked.title
+        if title == 'Fork of ' or title == '':
```

Comment:
Still have to handle components being forked. Component title would be an empty string ('').  With your suggestion, component title would be empty string, not None, which would make component title be empty string instead of the forked title.